exporter/kinesisexporter/config_test.go

```diff
@@ -56,20 +56,13 @@ func TestDefaultConfig(t *testing.T) {
 				FlushIntervalSeconds: 5,
 				MaxConnections:       24,
 			},
-
```

Comment:
Looking at this now, I wonder what we lose when we move to the omnition library. There seems to be some sharding logic that the opencensus-go-exporter is doing before sending it thru to the streams. Currently checking what's it doing.

Reply:
The sharding/paritition key logic in opencensus was done using the span ID, I replicated the behaviour within the ConsumeTraces function. The other parameters here seem to be parameters of the producer itself. 

I think we sacrifice fine grained control of the external library by switching to this.

Reply:
Yeah, I guess we could revisit whatever `shardProducer` logic they have once we're ready to submit this upstream. But for our current basic case, I'm good with it.